app/src/main/java/com/keylesspalace/tusky/adapter/StatusDetailedViewHolder.java

```diff
@@ -68,6 +74,47 @@ protected void setCreatedAt(@Nullable Date createdAt) {
         }
     }
 
+    private void setReblogAndFavCount(int reblogCount,  int favCount, StatusActionListener listener) {
+
+        NumberFormat numberFormat = NumberFormat.getNumberInstance();
```

Comment:
I would maybe reuse this format if possible

Reply:
good idea